spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/config/GoogleConfigPropertySourceLocatorTest.java

```diff
@@ -79,10 +79,10 @@ public void locateReturnsMapPropertySource() throws Exception {
 				this.projectIdProvider, this.credentialsProvider, this.gcpConfigProperties));
 		doReturn(googleConfigEnvironment).when(this.googleConfigPropertySourceLocator).getRemoteEnvironment();
 		PropertySource<?> propertySource = this.googleConfigPropertySourceLocator.locate(new StandardEnvironment());
-		assertEquals(propertySource.getName(), "spring-cloud-gcp");
-		assertEquals(propertySource.getProperty("property-int"), 10);
-		assertEquals(propertySource.getProperty("property-bool"), true);
-		assertEquals("projectid", this.googleConfigPropertySourceLocator.getProjectId());
+		assertThat(propertySource.getName()).isEqualTo("spring-cloud-gcp");
+		assertThat(propertySource.getProperty("property-int")).isEqualTo(10);
+		assertThat(propertySource.getProperty("property-bool")).isEqualTo(true);
+		assertThat("projectid").isEqualTo(this.googleConfigPropertySourceLocator.getProjectId());
```

Comment:
Flip arguments, please. You really expect that result of the execution equals to `"projectid"`, but not vise versa.